code/modules/projectiles/ammo_datums.dm

```diff
@@ -1931,6 +1931,41 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 /datum/ammo/xeno/boiler_gas/corrosive/set_smoke()
 	smoke_system = new /datum/effect_system/smoke_spread/xeno/acid()
 
+/datum/ammo/xeno/larvae_jelly
+	name = "blob of larvae jelly"
+	icon_state = "larvajelly"
```

Comment:
Probably don't need a whole new icon state for this, since it's just the same one as `sticky`.
```suggestion
	icon_state = "sticky"
```